examples/pipelineruns/workspaces-multiple-from.yaml

```diff
@@ -0,0 +1,195 @@
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: regional-disk
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: storage3
+spec:
+  resources:
+    requests:
+      storage: 16Mi
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: regional-disk
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: generate-number
+spec:
+  inputs:
+    params:
+    - name: num
+      default: "0"
+  workspaces:
+  - name: out
+  steps:
+  - name: write-num
+    image: ubuntu
+    script: echo "$(inputs.params.num)" > "$(workspaces.out.path)/number"
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: add-numbers
+spec:
+  workspaces:
+  - name: ws1
+    readOnly: true
+  - name: ws2
+    readOnly: true
+  - name: result
+  steps:
+  - name: calculate-sum
+    image: ubuntu
+    script: |
+      num1=$(cat $(workspaces.ws1.path)/number)
+      num2=$(cat $(workspaces.ws2.path)/number)
+      result=$(expr $num1 + $num2)
+      echo $result > $(workspaces.result.path)/number
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: check-number
+spec:
+  inputs:
+    params:
+    - name: expected
+  workspaces:
+  - name: in
+    readOnly: true
+  steps:
+  - name: check
+    image: ubuntu
+    script: |
+      input=$(cat $(workspaces.in.path)/number)
+      test "$input" -eq "$(inputs.params.expected)"
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: worlds-slowest-but-greatest-fibonacci
+spec:
+  workspaces:
+  - name: A
+    description: "A storage location to write numbers to"
+  - name: B
+    description: "Separate storage for numbers, for writing in parallel"
+  - name: C
+    description: "One more number storage location for parallel writes"
+  tasks:
+  - name: init-1
+    taskRef:
+      name: generate-number
+    params:
+    - name: num
+      value: "1"
+    workspaces:
+    - name: out
+      workspace: A
+  - name: init-2
+    taskRef:
+      name: generate-number
+    params:
+    - name: num
+      value: "1"
+    workspaces:
+    - name: out
+      workspace: B
+  - name: first-add
+    taskRef:
+      name: add-numbers
+    workspaces:
+    - name: ws1
+      from:
+        task: init-1
+        name: out
+    - name: ws2
+      from:
+        task: init-2
+        name: out
+    - name: result
+      workspace: C
+  - name: second-add
+    taskRef:
+      name: add-numbers
+    workspaces:
+    - name: ws1
+      from:
+        task: init-2
+        name: out
+    - name: ws2
+      from:
+        task: first-add
+        name: result
+    - name: result
+      workspace: A
+  - name: check-result
+    taskRef:
+      name: check-number
+    params:
+    - name: expected
+      value: "3"
+    workspaces:
+    - name: in
+      from:
+        task: second-add
+        name: result
+---
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  generateName: fib-
+spec:
+  pipelineRef:
+    name: horrible-fibonacci
```

Comment:
@sbwsg I think you need to change this to `worlds-slowest-but-greatest-fibonacci` 🤔

Reply:
this pipelinerun is failing with:

```
 kubectl get pipelinerun
NAME              SUCCEEDED   REASON               STARTTIME   COMPLETIONTIME
fib-pipelinerun   False       CouldntGetPipeline   26s         26s
```

Reply:
@sbwsg I might not have pulled all your code to test with (because of the conflict), please ignore it if you are not hitting this but the pods are stuck in pending: 

```
kubectl get pods
NAME                                     READY   STATUS    RESTARTS   AGE
fib-pipelinerun-init-1-5rglh-pod-zvznr   0/1     Pending   0          19m
fib-pipelinerun-init-2-nnq6v-pod-bq8xs   0/1     Pending   0          19m
```

with a message:

```
"message": "pod status \"PodScheduled\":\"False\"; message: \"pod has unbound immediate PersistentVolumeClaims\"",
```

Reply:
amazing, great catch.  I updated the pipelineref.

There are some remaining issues related to using regional PVCs in the example yaml.  I'm getting volume node affinity errors when I run it locally and it appears that similar issues are occurring here in the CI cluster as well.  Can't quite put my finger on what's going wrong but still looking into it.